pool/share.go

```diff
@@ -135,7 +135,7 @@ func pplnsEligibleShares(db *bolt.DB, min []byte) ([]*Share, error) {
 				return dbError(ErrDecode, desc)
 			}
 
-			if bytes.Compare(createdOnB, min) > 0 {
+			if bytes.Compare(createdOnB, nanoToBigEndianBytes(min)) > 0 {
```

Comment:
same here `nanoToBigEndianBytes` is going to be called once each comparison when it could be done just one outside the loop.